### DISCUSSION TITLE
Trouble building latest 'top-of-the-trunk' global workflow

### DISCUSSION THREAD

**Discussion by mdtoyNOAA** [NOAA-EMC/global-workflow#1909] [Q&A] [answered]
Back in January I was able to clone, build and run the global workflow using the online Wiki instructions at:
1) [clone and build](https://github.com/NOAA-EMC/global-workflow/wiki/Clone-and-build-global-workflow/08b2ec4d0956ea1936d567d9a1ac32614aedfec1)
2) [run setup scripts](https://github.com/NOAA-EMC/global-workflow/wiki/Run-setup-scripts-to-generate-experiment/c85d5c2f87b298b6f0536a2fc2fb425a092c70fc)

Last week I pulled the latest changes to the repo and wasn't able to get the workflow to work anymore.  I decided to start from the beginning and clone and build the workflow from scratch, but I've been unsuccessful.  It seems the options for the 'setup' scripts have changed over time.

The following line for 'setup_expt.py' used to work (on Hera):
./setup_expt.py forecast-only --idate 2019110300 --edate 2020032700 --app ATM --start cold --gfs_cyc 1 --resdet 768 --pslot GFS_v17_exp0 --comrot /scratch1/BMC/wrfruc/mtoy/git_local/global-workflow.Oct_2023/FV3GFSrun --expdir /scratch1/BMC/wrfruc/mtoy/git_local/global-workflow.Oct_2023/FV3GFSwfm

Now I get the following error message:
usage: setup_expt.py [-h] {gfs,gefs} ...
setup_expt.py: error: argument system: invalid choice: 'forecast-only' (choose from 'gfs', 'gefs')

It looks like the required arguments have changed.  Please let me know if there is any updated documentation on how to run 'setup_expt.py' and 'setup_workflow.py'.  Thank you.

**Comment by WalterKolczynski-NOAA** [comment]
Documentation has moved to [ReadTheDocs](https://global-workflow.readthedocs.io/en/latest/). It should be up-to-date there.

**Reply by mdtoyNOAA** [reply, marked as answer]
Thank you @WalterKolczynski-NOAA.  I will look there.

**Comment by aerorahul** [comment]
I think this is resolved.
